src/coreclr/src/gc/gc.cpp

```diff
@@ -35884,7 +35686,9 @@ gc_heap::verify_heap (BOOL begin_gc_p)
     BOOL            large_brick_p = TRUE;
     size_t          curr_brick = 0;
     size_t          prev_brick = (size_t)-1;
-    int             curr_gen_num = max_generation+1;
+    
+    // go through all generations starting with loh
```

Comment:
would be better to start with (total_generation_count - 1) here - inevitably we'd want to verify all generations.